recipe/build.sh

```diff
@@ -2,14 +2,18 @@
 
 set -e
 
-test -d m4 || mkdir m4
-# gtkdocize || exit 1
+echo "DIAGNOSTIC environment:"
+env | sort
 
-export ACLOCAL_PATH="$PREFIX/share/aclocal"
-aclocal --print-ac-dir
+echo "DIAGNOSTIC directory listing:"
+ls -lAFgh
 
-autoreconf -ivf
+echo "DIAGNOSTIC pwd: $(pwd)"
 
-./configure --disable-static --disable-gui --enable-introspection=yes --disable-hkl-doc --prefix=$PREFIX
-make -j ${CPU_COUNT}
-make install
+TARBALL="libhkl-${PKG_VERSION}-x86_64.tar.gz"
```

Comment:
Need to use the same version as downloaded.
```suggestion
TARBALL="libhkl-v${PKG_VERSION}-x86_64.tar.gz"
```